src/pamiq_core/threads/thread_types.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum, auto
+
+
+class ThreadTypes(Enum):
```

Comment:
docstringを追加しましょう。これは一体何のためにあるのか、簡潔に記述します。
```suggestion
class ThreadTypes(Enum):
    """Declares all thread types."""
```

Reply:
Docstring 忘れておりました、、、ありがとうございます！　修正します！